README.md

```diff
@@ -52,6 +54,18 @@ DogWatch.monitor do
   end
 end
 ```
+Queries are the combination of several items including a time aggregator (over a window of time), space aggregator (avg, sum, min, or max), a set of tags and an optional order by, and an operator.
+
+From the Datadog documentation:
```

Comment:
@eciramella-r7 it'd be worthwhile linking the documentation if they have more information on query strings IMO.